src/backend/apidb/changeset_upload/relation_updater.cpp

```diff
@@ -1353,10 +1353,10 @@ void ApiDB_Relation_Updater::save_current_relations_to_history(
 
   m.prepare("current_relations_to_history",
             R"(   
-                INSERT INTO relations 
-                (SELECT id AS relation_id, changeset_id, timestamp, version, visible
-                 FROM current_relations
-                 WHERE id = ANY($1)) 
+                INSERT INTO relations (relation_id, changeset_id, timestamp, version, visible)
+                SELECT id AS relation_id, changeset_id, timestamp, version, visible
```

Comment:
In the current_relations_to_history query below, we use an alias as an `SELECT id AS relation_id`, whereas current_nodes_to_history and current_ways_to_history don't use an alias `SELECT id,`.

It doesn't really make a difference here, it's just something I noticed while looking through the changes.